.yamato/upm-ci.yml

```diff
@@ -98,12 +98,41 @@ test_trigger:
         - "upm-ci~/packages/**/*"
   dependencies:
     - .yamato/upm-ci.yml#pack
-    {% for editor in test_editors_all %}
-    {% for platform in test_platforms %}
+      {% for editor in test_editors_all %}
+      {% for platform in test_platforms %}
     - .yamato/upm-ci.yml#test_{{platform.name}}_{{editor.version}}
-    {% endfor %}
-    {% endfor %}
+      {% endfor %}
+      {% endfor %}
```

Comment:
Minor formatting comment: if we're at fixing the format of this file I would follow the way it was made in "run_preview_verified_staging" job, i.e. keep the jinja template stuff with no indentation as it's something apart from the regular yaml file and so it has no reason to follow the same yaml indentation (otherwise we could argue that nested for loops should have different indentation, but in some case it would cause problem, blah blah blah). Also it will be less changes that are not related to the purpose of the PR.

But I'm aware that there is no known official convention (or maybe there is and I don't know it) and so that's a "user preference" thing.

Reply:
Yeah I was checking the one in the recorder repo and given there was 2 different styles in the yamato usd file I used the one from recorder.
After a chat with @clusty we agreed on aligning to either the - or the . of the line before so that it doesn't break the flow and looks weird when more complex liquid instructions with indentations are needed.